benefit/VERSION.yaml

```diff
@@ -1 +1 @@
-TAG: 5.15.0
+TAG: S_10167-RC-4
```

Comment:
Will need to be set to the appropriate version and in CaseType

Reply:
I think I have done this but, you may want a larger change in value.